zalando/zalando.go

```diff
@@ -21,14 +21,18 @@ var AccessTuples []AccessTuple
 // AccessTuple is the type defined for use in AccessTuples.
 type AccessTuple struct {
 	Realm string `yaml:"realm,omitempty"` // p.e. "employees", "services"
-	Uid   string `yaml:"uid,omitempty"`   // UnixName
-	Cn    string `yaml:"cn,omitempty"`    // RealName
+	//lint:ignore ST1003 public interface
```

Comment:
I think that disabling is setting a bad example here? Better to fix the linter violations here.